client/src/app/site/modules/user-components/components/user-detail-view/user-detail-view.component.html

```diff
@@ -159,6 +159,14 @@ <h2>{{ 'Personal information' | translate }}</h2>
             </mat-form-field>
         </div>
 
+        <div *ngIf="isAllowed('update')">
+            <mat-form-field class="width-16p force-min-width">
+                <mat-label>{{ 'Membership number' | translate }}</mat-label>
+                <input type="text" matInput formControlName="member_number" />
+                <mat-error>{{ 'Membership number must be unique' | translate }}</mat-error>
```

Comment:
This Error is never activated.
If possible fix the activation 
If not possible please remove the line

Reply:
I have removed this mat-error. The uniqueness check is done in the backend. See username eg.